ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -359,14 +359,14 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "c21Orders",
+    "CaseFieldID": "generatedOrders",
```

Comment:
What about `orderCollection`?

Reply:
Yeah I like that better. I'll make the change I want to try and use 'generated' as little as possible, because really it should just be 'Order'... Might ask around for more ideas of what to call that class